lib/rack/prx_auth.rb

```diff
@@ -53,7 +53,12 @@ def decode_token(token)
     end
 
     def expired?(claims)
-      Time.now.to_i > (claims['iat'] + claims['exp'])
+      now = Time.now.to_i
+      now < claims['iat'] || if claims['iat'] <= claims['exp']
```

Comment:
I find this a bit hard to read with the indentation ... maybe just ye olde if/elsif/else?